utils/helpers.ts

```diff
@@ -151,3 +153,56 @@ export function checkSearchTerm(searchTerm: string, plant: Plant) {
   // Check if plant_name contains searchTerm
   return plant.plant_name.toLowerCase().includes(searchTerm.toLowerCase());
 }
+
+export function fillCalendarGridArrayRowWithColor(
+  startMonth: string,
+  endMonth: string,
+  color: string,
+  rowIndex: number,
+  gridArray: string[],
+) {
+  // if startMonth and endMonth is both null, row should be empty
```

Comment:
Earlier, I was confused on when this would be the case

maybe we can also include in the comment that this case occurs e.g. when a plant cannot be planted indoors (i.e. indoors_start and indoors_end are both null)